pegasus/sites.v3/code.org/public/learn/index.haml

```diff
@@ -42,8 +42,10 @@ social:
         %br/
       #studentimage.col-50.tablet-feature{style: "height: 278px"}
 
--# Not ideal but we can pull the undigested file from /blockly.
-%script{src: CDO.studio_url('/blockly/js/tutorialExplorer.js')}
+-# Not ideal but we can pull the undigested files from /blockly.
+- js_locale = request.locale.to_s.downcase.tr('-', '_')
```

Comment:
What do you mean? Like whether this is the proper way to get the locale for all locales?